brave/brave-core
# Refactored Brave Ads URL helper and added unit tests

Resolves https://github.com/brave/brave-browser/issues/9356

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Confirm viewing an ad
- Confirm clicking an ad
- Confirm purchase intent
- Confirm ad conversions

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
